third_party/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if(NOT VELOX_DISABLE_GOOGLETEST)
 endif()
 
 add_subdirectory(xsimd)
+add_subdirectory(simdjson)
```

Comment:
Please don't add it as a submodule. Please check out our cmake dependency system in `CMake/resolve_dependency_modules`. There is a readme and it should be pretty straight forward to add simdjson.

Reply:
https://github.com/facebookincubator/velox/tree/main/CMake/resolve_dependency_modules